runtime/rasdump/trigger.c

```diff
@@ -386,7 +386,7 @@ matchesExceptionFilter(J9VMThread *vmThread, J9RASdumpEventData *eventData, UDAT
 				}
 			} else {
 				/* For other events, walk the stack to find the desired frame */
-				vmThread->javaVM->internalVMFunctions->iterateStackTrace(vmThread, (j9object_t*) eventData->exceptionRef, countExceptionStackFrame, &throwSite, TRUE);
+				vmThread->javaVM->internalVMFunctions->iterateStackTrace(vmThread, (j9object_t*) eventData->exceptionRef, countExceptionStackFrame, &throwSite, TRUE, FALSE);
```

Comment:
I'm not sure if this one should always be false. @keithc-ca

Reply:
It might even make sense for it to always skip hidden frames.

Reply:
Need a decision here before we merge.

FALSE means you always have to know the exact shape of the stack to trigger your event.

TRUE means the trigger will be based on the view of the stack unaffected by implementation details.

I'm not sure we should change the trigger behaviour based on the show hidden frames command line option. I personally would lean towards always using TRUE here.

Reply:
just to be more clear, TRUE means always skip hidden frames, FALSE means do not skip hidden frames.
the cmdline option would only affect this if we use that to determine this value as TRUE or FALSE. as done in `jclexception.cpp`

Reply:
I think hidden frames should not be skipped (the new argument should be `FALSE`) to retain existing behavior of being able to use filters that refer to hidden frames.

Reply:
It may be worth explicitly documenting the fact that the command line option will not affect the trigger.

Reply:
Please open a doc issue or PR for this before merge.

Reply:
Opened issue https://github.com/eclipse-openj9/openj9-docs/issues/910